tailwind-colors.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import resolveConfig from "tailwindcss/resolveConfig";
+
+import tailwindConfig from "./tailwind.config";
+
+// $FlowIgnore
+const fullConfig = resolveConfig( tailwindConfig );
+
+export default fullConfig.theme.colors;
```

Comment:
FYI, our convention is for camelcase file names, so if you were to use this approach, I would name this file `tailwindColors`, not `tailwind-colors`.